.gitattributes

```diff
@@ -1 +1,2 @@
 docs/* linguist-vendored
+docs/criterion/** linguist-generated=true -diff
```

Comment:
How is it different from `linguist-vendored` and what does `-diff` do?

Reply:
Also I'm not sure why you would request changes based on the question? Which changes do I need to make?

Reply:
https://github.com/github/linguist#vendored-code
https://github.com/github/linguist#generated-code
https://git-scm.com/docs/gitattributes#_marking_files_as_binary

Reply:
I somehow failed to duckduckgo this info. It should've been a comment instead of request for changes.